methods.tex

```diff
@@ -34,16 +33,16 @@ \subsection{Performance Measurement}
   E = \frac{S}{N}.
 \end{equation}
 Additionally, we introduced two performance parameters that we will show are indicative of the occurrence of stragglers.
-We define the ratio of compute time to I/O time $t_{\text{Compute}}/t_{\text{IO}}$. 
-In the present study, we calculated this ratio using the serial version of our algorithms because compute time per frame and I/O time per frame should ideally be the same for different runs using different processes, as we observed in our previous study \cite{Khoshlessan:2017ab}.
+We define the ratio of compute time to I/O time as $t_{\text{Compute}}/t_{\text{IO}}$. 
+In the present study, we calculated this ratio using the serial version of our algorithms because compute time per frame and I/O time per frame should ideally be the same for different runs using different processes, as we have observed previously~\cite{Khoshlessan:2017ab}.
 The ratio of compute to communication time is defined by the ratio of total compute time per rank averaged across all ranks to the total communication time per rank averaged across all ranks 
 \begin{gather}
   \label{eq:Compute-comm}
-  \frac{t_{\text{comp}}}{t_{\text{comm}}} = \frac{\frac{1}{N}
+  \frac{\overline{t_{\text{comp}}}}{\overline{t_{\text{comm}}}} = \frac{\frac{1}{N}
```

Comment:
The overline that you left here makes it confusing. We call the these definition tcomp/tcomm and we use it everywhere in the text which corresponds to this definition. There is no need for having overline here.

Reply:
Yes, you are right. I need to correct this in all parts of the text.

Reply:
Can you wait until I resolve the conflict and use this branch instead of master? It will avoid conflicts and it will make things easier.

Reply:
Now it is ready and it is conflict free!

Reply:
I corrected it in the text for everything.

Reply:
Is the correction in the master branch or here? I do not see a commit in the PR.